server.py

```diff
@@ -35,29 +41,33 @@ def log_reponse_info(response):
 
 # protected/non-protected routes
 open_routes = {
-    ('/login', 'POST'): 'login',
-    ('/register', 'POST'): 'register',
-    ('/create_bank_account', 'POST'): 'create_bank_account'
+    ('/users/login', 'POST'): 'login',
+    ('/users/register', 'POST'): 'register',
+    ('/bank_accounts', 'POST'): 'create_bank_account'
```

Comment:
Why create_bank_account is open? Don't you need the user_id of the account owner?